default-views/account/register.hbs

```diff
@@ -21,5 +21,57 @@
     {{> account/register-form}}
   {{/if}}
 </div>
+<script src="https://raw.githubusercontent.com/nowsecure/owasp-password-strength-test/master/owasp-password-strength-test.js"
+        defer></script>
+<script>
+    function validatePasswordBeforeSubmit (e) {
+        e.preventDefault();
+        const pwdErrorDiv = document.getElementById('passwordHelp');
```

Comment:
This is a not an ID that is present in the markup yet? (I cannot find it, at least)

Reply:
Looks like I forgot to add a file to the PR. Will do it asap.